docs/pages/world/reference/internal/init-module-implementation.mdx

```diff
@@ -475,6 +475,18 @@ function registerDelegation(
 | `delegationControlId` | `ResourceId` | The ID controlling the delegation          |
 | `initCallData`        | `bytes`      | The initialization data for the delegation |
 
+#### registerDelegationWithSignature
+
+```solidity
+function registerDelegationWithSignature(
+  address delegatee,
+  ResourceId delegationControlId,
+  bytes memory initCallData,
+  address delegator,
+  bytes memory signature
```

Comment:
```suggestion
  bytes calldata signature
```

Uses slightly less gas and doesn't change the functionality. `calldata` can be used on params that do not get altered inside the function.